web/Services/AdminService.cs

```diff
@@ -51,4 +53,12 @@ public async Task<(List<UserDbModel> CurrentAdmins, List<UserDbModel> PendingAdm
     {
         return (await GetCurrentAdmins(), await GetPendingAdmins());
     }
+
+    public void ApproveAdminAccessRequest(int userId)
+    {
+        var userToAdmin = _dbContext.Users.Single(u => u.Id == userId);
```

Comment:
In general it's preferable to use the async method where possible. Additionally `SingleOrDefault` is safer as it handles the case where the user ID doesn't exist 
```suggestion
var userToMakeAdmin = await _dbContext.Users.SingleOrDefaultAsync(u => u.Id == userId);
```

Reply:
![image](https://user-images.githubusercontent.com/84274691/181225757-a7c964fc-b530-4d01-a02a-47022e445f75.png)
how can I change this?